src/Sulu/Bundle/AdminBundle/Resources/js/containers/MultiSelection/tests/MultiSelection.test.js

```diff
@@ -165,28 +188,85 @@ test('Pass itemDisabledCondition to MultiListOverlay', () => {
     expect(selection.find('MultiListOverlay').prop('itemDisabledCondition')).toEqual('status == "inactive"');
 });
 
-test('Show with passed values as items in right locale', () => {
+test('Construct MultiSelectionStore with correct parameters', () => {
     const locale = observable.box('en');
+    const options = {key: 'value-1'};
```

Comment:
I think I would not extract that into an own variable... The `MultiSelection` component could falsely manipulate that object, and then the check below would still work.